### ISSUE TITLE
Quality of metadata issues related to fwp* (Windows Filtering Platform)

### ISSUE THREAD

**Issue by JesperTreetop** [microsoft/win32metadata#191]
[`FWPM_DISPLAY_DATA0`](https://docs.microsoft.com/en-us/windows/win32/api/fwptypes/ns-fwptypes-fwpm_display_data0) contains two wide strings:

```c
typedef struct FWPM_DISPLAY_DATA0_ {
  wchar_t *name;
  wchar_t *description;
} FWPM_DISPLAY_DATA0;
```

and is being projected into winmd as:

```csharp
public struct FWPM_DISPLAY_DATA0 {
    public ushort* name;
    public ushort* description;
}
```

`ushort` is calling-convention/PInvoke compatible, but useful information is clearly being dropped. There should be a way to encode (extra attributes?) that this is meant to be a wide string pointer so that CsWin32 and the like can produce better interfacing code.

**Comment by JesperTreetop** [comment]
This looks like it's related to #153.

**Comment by AArnott** [comment]
I would say it's more related to #59

**Comment by sotteson1** [comment]
Fixed with 10.0.19041.5-preview.5.
